Backport this 3g modem that I have.
Given it's quite common, should be useful
And I hate when I forget to patch the kernel after a re-emerge.

--- sys/dev/usb/usbdevs.orig
+++ sys/dev/usb/usbdevs
@@ -2055,6 +2055,7 @@
 product QUALCOMM2 CDMA_MSM	0x3196	CDMA Technologies MSM modem
 product QUALCOMMINC CDMA_MSM	0x0001	CDMA Technologies MSM modem
 product QUALCOMMINC ZTE_STOR	0x2000	USB ZTE Storage
+product QUALCOMMINC MF626	0X0031	3G modem
 product QUALCOMMINC AC8700	0xfffe	CDMA 1xEVDO USB modem
 
 /* Qtronix products */
--- sys/dev/usb/serial/u3g.c.orig
+++ sys/dev/usb/serial/u3g.c
@@ -184,6 +184,7 @@
 	/* OEM: Qualcomm, Inc. */
 	U3G_DEV(QUALCOMMINC, ZTE_STOR, U3GFL_SCSI_EJECT),
 	U3G_DEV(QUALCOMMINC, CDMA_MSM, U3GFL_SCSI_EJECT),
+	U3G_DEV(QUALCOMMINC, MF626, 0),
 	/* OEM: Huawei */
 	U3G_DEV(HUAWEI, MOBILE, U3GFL_HUAWEI_INIT),
 	U3G_DEV(HUAWEI, E180V, U3GFL_HUAWEI_INIT),
